RELEASE_PROCESS.md

```diff
@@ -22,8 +22,8 @@ In order to release, you will need the following accounts/permissions:
 - Local checkouts of the chef-server and chef-web-downloads repositories
 - Push access to the chef-server github repository
 - Chef Software, Inc Slack account
-- blog.chef.io wordpress account
 - VPN account for Chef Software, Inc.
+- Account on [https://discourse.chef.io/](https://discourse.chef.io/) using your Chef email address
 - Login for wilson.ci.opscode.us (This is linked to your github
```

Comment:
Yeah, it's github. It uses the github oauth system. Eng Services should have full details.